src/controllers/media.ts

```diff
@@ -39,7 +39,7 @@ const getFromIMDbAPI = async(imdbId?: string, searchRequest?: SearchRequest): Pr
    */
   if (!imdbId) {
     const parsedFilename = episodeParser(searchRequest.name);
-    const isTVEpisode = parsedFilename && parsedFilename.show && parsedFilename.season && parsedFilename.episode;
+    const isTVEpisode = parsedFilename?.show && parsedFilename.season && parsedFilename.episode;
```

Comment:
Wouldn't it need it on the season and episode too? Or maybe I misunderstand optional chaining

Reply:
Yeah you are right. There is little value even changing this line tbh. Maybe if the chain of property access was longer than 2